### PR TITLE
Remove groupId already specified in parent pom

### PR DESCRIPTION
**Pull request by gerashegalov**
Signed-off-by: Gera Shegalov <gera@apache.org>

Minor maven cleanup. Contributes to #2109

### CODE REVIEW

**Comment by gerashegalov** [comment]
build

File: shims/spark320/pom.xml [hunk 1]
```diff
@@ -25,8 +25,7 @@
     <version>0.5.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <groupId>com.nvidia</groupId>
-  <artifactId>rapids-4-spark-shims-spark320_2.12</artifactId>
+    <artifactId>rapids-4-spark-shims-spark320_2.12</artifactId>
```
Comment (jlowe) [inline shims/spark320/pom.xml:28]: Misindented line here.

File: udf-examples/pom.xml [hunk 2]
```diff
@@ -24,8 +24,7 @@
     <artifactId>rapids-4-spark-parent</artifactId>
     <version>0.5.0-SNAPSHOT</version>
   </parent>
-  <groupId>com.nvidia</groupId>
-  <artifactId>rapids-4-spark-udf-examples_2.12</artifactId>
+    <artifactId>rapids-4-spark-udf-examples_2.12</artifactId>
```
Comment (jlowe) [inline udf-examples/pom.xml:27]: Misindented line here.

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Pull request merged**